resources/lib/kodi/library.py

```diff
@@ -363,7 +397,11 @@ def remove_item(item_task, library_home=None):
         except Exception:
             common.debug('Cannot delete {}, file does not exist'
                          .format(exported_filename))
-        common.remove_path(id_path, g.library(), lambda e: e.keys() == ['videoid'])
+
+        # Is there a way to simplify the lambda expression ?
+        common.remove_path(id_path, g.library(), lambda e: (e.keys() == ['videoid']
+                                                            or e.keys() == ['videoid', 'nfo_export']
+                                                            or e.keys() == ['nfo_export', 'videoid']))
```

Comment:
you can try to replace the two line 403-404 with:
`or all(k in e.keys() for k in ['videoid', 'nfo_export'])`

Reply:
or len(set(e.keys()) - {'videoid', 'nfo_export'}) == 0 works!